src/Adapters/Storage/Mongo/MongoTransform.js

```diff
@@ -565,9 +663,22 @@ function transformConstraint(constraint, field) {
     case '$gte':
     case '$exists':
     case '$ne':
-    case '$eq':
+    case '$eq': {
```

Comment:
we may enter on some weird expectations here: `$ne { $relativeTime: "1 day ago" }` would you expect all object that were created the full day before to be excluded or just the one that matches perfectly the current time?

Reply:
@montymxb what do you think? Should we limit to $lt/$lte/$gt/$gte to be safe?

Reply:
Hmm, it would make the most sense with the query methods you've mentioned above, but not sure if we should simply rule out $ne/$eq. Although they may not be the most effective query methods with relative date matching they _could_ still work, especially if we allow setting the `now` date in the future (but still not that great).

I would just say exclude those, but if someone expects it to work what would we return? Just an empty response like the query matched nothing, or provide some sort of error clarifying that equal/notEqual ops are not allowed using this param, or another approach perhaps.

Reply:
Or throw an error, that relativeDates operators don’t work with those operators

Reply:
That would work for me 👍 . Simplifying the initial implementation may be best, we can consider it later if someone actually needs to use it.